neural_tangents/predict.py

```diff
@@ -138,8 +137,7 @@ def gradient_descent_mse(
 
   @lru_cache(1)
   def get_predict_fn_inf():
-    with jax.core.eval_context():
-      solve = _get_cho_solve(k_train_train, diag_reg, diag_reg_absolute_scale)
+    solve = _get_cho_solve(k_train_train, diag_reg, diag_reg_absolute_scale)
```

Comment:
Does the LRU caching work with tf.function?

Reply:
This is a point that I do not know...What I did is just getting rid of the `jax.core` dependency @wangpengmit, @allenlavoie, @akshaym Would you mind elaborating on this point that Sam mentioned? Thanks!